content/v2.0/router/overview.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Router: Overview"
+order: 10
+---
+
+# Overview
+
+Hanami applications use [Hanami::Router](https://github.com/hanami/router) for routing: a Rack compatible, lightweight and fast HTTP router for Ruby.
+
+## The first route
+
+With your favorite editor open `/config/routes.rb` and add the following line inside of the `main` slice block.
+
+```ruby
+get '/hello', to: ->(env) { [200, {}, ['Hello from Hanami!']] }
+```
+
+So the whole file would look like this:
+
+```ruby
+# /config/routes.rb
+
+Hanami.application.routes do
+  slice :main, at: "/" do
+    root to: "home.show"
+
+    get '/hello', to: ->(env) { [200, {}, ['Hello from Hanami']] }
+  end
+end
+```
+
+Then start the server with `puma config.ru` and visit [http://localhost:3000/hello](http://localhost:2300/hello).
+You should see `Hello from Hanami!` in your browser.
+
+<p><img src="/v2.0/router/hello-from-hanami.png" alt="Hanami welcome page" class="img-responsive"></p>
+
+Let's explain what we just did.
+We created a **route**; an application can have many routes. By doing this, we told our app, to handle HTTP `get` request under the path: `/hello`, and handle it by calling a `Proc` object, that returns the status: `200`, empty headers, and the `'Hello from Hanami'` body. This response is given back to the browser for rendering.
```

Comment:
```suggestion
We created a **route**; an application can have many routes. By doing this, we told our app to handle HTTP `get` request under the path `/hello`, and handle it by calling a lambda that returns the status code `200` (a success), with empty headers, and the `'Hello from Hanami'` body. This response is given back to the browser for rendering.
```
A lambda is an instance of `Proc`, but they behave differently. Since Hanami::Router is agnostic about which it received (I assume 😇 ), I think saying that it's a `Proc` might make people think it wants non-lambda Procs

Reply:
Lol. Didn't know! :D Thanks! By writing these docs I'll learn more advanced Ruby's tips, than in a year of regular development ;).